go/coordinator/internal/metastore/db/dao/segment_metadata.go

```diff
@@ -21,7 +21,7 @@ func (s *segmentMetadataDb) DeleteBySegmentID(segmentID string) error {
 func (s *segmentMetadataDb) DeleteBySegmentIDAndKeys(segmentID string, keys []string) error {
 	return s.db.
 		Where("segment_id = ?", segmentID).
-		Where("`key` IN ?", keys).
+		Where("key IN ?", keys).
```

Comment:
Reason for change?

Reply:
because unit tests were using sqlite and this throws an error when I change them to postgres